code/controllers/subsystem/air.dm

```diff
@@ -124,11 +196,8 @@ var/datum/subsystem/air/SSair
 						T.excited = 1
 						active_turfs |= T
 
-/datum/subsystem/air/proc/process_excited_groups()
-	for(var/datum/excited_group/EG in excited_groups)
-		EG.breakdown_cooldown ++
-		if(EG.breakdown_cooldown == 10)
-			EG.self_breakdown()
-			return
-		if(EG.breakdown_cooldown > 20)
-			EG.dismantle()
\ No newline at end of file
+/datum/subsystem/air/proc/setup_atmos_machinery(z_level)
```

Comment:
So the diffs are showing something fucking weird.

This is not in setup all turfs. this is in its own proc called setup_atmos_machinery. If you click view file, thats what you see. no idea why github is showing this diff so odd. and more importantly, no idea what this file will look like if this pr gets merged. what the fuck git.